apps/cyberstorm-nextjs/middleware.ts

```diff
@@ -24,6 +24,12 @@ export async function middleware(request: NextRequest) {
   const req = request;
   const response = NextResponse.next();
 
+  if (process.env.NEXT_PUBLIC_PRODUCTION_PAGE_STRUCTURE) {
+    if (request.nextUrl.pathname.startsWith("/communities")) {
```

Comment:
This blocks the access to all subpages under `/communities`. Sure, we don't have them yet, but wouldn't hurt to use exact matching here either.